extension/intents/forms/formsContentScript.js

```diff
@@ -176,6 +176,11 @@ this.dictationContentScript = (function() {
     return focusDirection(1);
   });
 
+  communicate.register("formSubmit", async message => {
+    document.activeElement.form.submit();
```

Comment:
This is looking pretty good, though it should also check and give an error here: sometimes `document.activeElement.form` will be null. Not infrequently actually, anytime the active element isn't a form element this will be true. So for instance if you click on the background of the page and then try to submit a form you'll get this:

<img width="411" alt="image" src="https://user-images.githubusercontent.com/44390/76128904-4c389d80-5fcb-11ea-9942-867524c54f10.png">

If you look for `displayMessage` you'll see the pattern to throw a special exception that will give the user an error message. In this case the user should see something like "Please focus on an input field before trying to submit"